config/manager/manager-service.yaml

```diff
@@ -3,11 +3,15 @@ kind: Service
 metadata:
   name: service
   labels:
-    control-plane: controller-manager
-    app.kubernetes.io/created-by: data-science-pipelines-operator
+    app.kubernetes.io/name: service
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/part-of: data-science-pipelines-operator
+    app.kubernetes.io/instance: controller-manager
```

Comment:
This should have some part of unique id per installation, so that should probably be left to the deployer.

Reply:
By unique id do you mean a version id? What could the version id be based on? 

If this would be something based on versions of DSP, then I think we would need to update it for every version. Is this label really required then?

Reply:
Not version id, because if we follow the conventions of that label, it should be distinct between two different installation (even of the same version). That's why I think we should scrap it out of the yaml, only the installer has the capacity to generate and apply a random id for each instance (maybe that's doable with kustomize generator stuff, but I'm not too sure)